recipe/meta.yaml

```diff
@@ -72,3 +71,5 @@ extra:
     - sinhrks
     - tomaugspurger
     - jrbourbeau
+  skip-lints:
```

Comment:
This suppresses complaints from our linter?

Reply:
Yes, in this case it's for `packaging`. I already fixed the linter so that it stops complaining when `packaging` is found in the runtime dependencies. I guess the linter wasn't updated in prefect.